src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -146,7 +148,9 @@ const ConfirmTransactionModal = ({
                 <Heading tag="h3">Value</Heading>
                 <div className="value-section">
                   <Img alt="Ether" height={40} src={getEthAsToken('0').logoUri} />
-                  <Bold>{humanReadableValue(tx.value, 18)} ETH</Bold>
+                  <Bold>
+                    {humanReadableValue(tx.value, 18)} {nativeCoin.name}
```

Comment:
can we replace 18 with decimals amount from config?

Reply:
:point_up: and use `fromTokenUnit` instead.